src/Post_Command.php

```diff
@@ -424,7 +424,7 @@ public function list_( $_, $assoc_args ) {
 	 * default:
 	 * ---
 	 *
-	 * [--post_date=<yyyy-mm-dd>]
+	 * [--post_date=<yyyy-mm-dd hh:ii:ss>]
```

Comment:
This should actually be `<yyyy-mm-dd-hh-ii-ss>`, see https://github.com/wp-cli/entity-command/pull/43